examples/transfer/workflow.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: transfer.aws.upbound.io/v1beta1
+kind: Workflow
+metadata:
+  annotations:
+    meta.upbound.io/example-id: transfer/v1beta1/workflow
+    upjet.upbound.io/manual-intervention: "This resource manual intervention due to Uptest failed without any errors."
```

Comment:
Could you please link in the PR to the failed attempt as a reference?

Reply:
Added link to description.

Reply:
We took a look at the logs referenced above with @turkenf but it looks like it was a temporary issue. If we have not already retried this more than once, could you please give it a retry (after deleting the manual-intervention annotation)?